docs/source/user/configuration.rst

```diff
@@ -72,6 +73,15 @@ to be in the ``flake8`` section, which means it needs to start like so:
 
     [flake8]
 
+.. note::
+
+   If your project does not use ``setup.cfg``, ``tox.ini``, neither
+   ``.flake8``, or they don't contain section ``flake8`` at all, the searching
+   for suitable configuration file will continue in project's parent
+   directories. This may lead to parsing no longer supported user-level
+   configuration at ``~/.flake8``. Please do not rely on this behaviour hence
+   we are supporting only per-project configuration.
+
```

Comment:
I'd rather not document an unsupported behaviour

Reply:
So just leave out the last two sentences? The beginning part is supported behaviour, isn't it?